app/mailers/mailer.rb

```diff
@@ -78,6 +78,11 @@ def not_enough_users_assigned(task, user)
   def mail(args)
     args[:subject] = "[#{FoodsoftConfig[:name]}] #{args[:subject]}"
 
+    if args[:from].is_a? User
+      args[:reply_to] = args[:from] unless args[:reply_to]
+      args[:from] = "#{show_user args[:from]} via #{I18n.t('layouts.foodsoft')} <#{FoodsoftConfig[:email_sender]}>"
```

Comment:
i18n!

Reply:
(hmm now I see it was like this already, feel free to add or keep)

Reply:
I'll address it in a separate issue then the locale is out of the plugin directory...)